docs/sources/writing-guide/reuse-shared-content.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Reuse shared content
+description: How to reuse chunks of content between pages
+keywords:
+  - content reuse
+  - shared content
+---
+
+# Reuse shared content
+
+Shared content is a sentence, paragraph, or whole page of text that makes sense in multiple contexts.
+Reusing shared content propagates changes from a single source file to one or more destination pages.
+
+This topic describes how to extract and share a chunk of content to multiple pages.
+
+## Before you begin
+
+- Identify a chunk of content that should be reused in multiple pages.
+
+## Steps
+
+To reuse shared content:
+
+1. Identify the sharing project.
+
+   If the shared content is to be reused in a single project, then the sharing project is that project.
+   However, if shared content is to be reused across multiple projects, for example Tempo and Grafana Enterprise Traces, then you must choose which project shares the content.
+
+   Prefer sharing from open source projects.
+
+1. In the sharing project, create the `docs/sources/shared/` directory if it does not exist.
+
+1. In the sharing project, create the file `docs/sources/shared/index.md` with the following contents:
+
+   ```markdown
+   ---
+   headless: true
+   ---
+   ```
+
+   The `index.md` file tells Hugo that the `docs/sources/shared` directory is a leaf bundle which is necessary for the `docs/shared` shortcode to access files stored within.
+
+   For more information about leaf bundles, refer to [Pages and page bundles]({{< relref "./documentation-structure/#pages-and-page-bundles" >}}).
```

Comment:
general comment, it's hard to paint the picture of what these steps look like. I would be inclined to add a screenshot that shows what we want and to add an explanation.

Reply:
Added in https://github.com/grafana/writers-toolkit/pull/189/commits/51ebb9701a796b2aad50b64e69c8e1fb607bad29